bot/voice.go

```diff
@@ -0,0 +1,92 @@
+package bot
+
+import (
+	"log"
+	"strings"
+	"time"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+func dcCmd(ctx CommandCtx) error {
+	if ctx.Message.Author.ID == morthisId {
+		var uId string
+
+		if len(ctx.Args) < 1 {
+			uId = venovaId
+		} else {
+			uId = strings.Trim(ctx.Args[0], "<@>")
+		}
+
+		if uId == venovaId {
+			stopAudio()
+			time.Sleep(500 * time.Millisecond)
+		}
+		err := disconnectUserFromVC(ctx.Session, ctx.Message.GuildID, uId)
+		if err != nil {
+			return ctx.Reply("Error disconnecting from voice channel.")
+		}
+		if uId == venovaId {
+			return ctx.Reply("Disconnected Venova from the server.")
+		} else {
+			return ctx.Reply("Disconnected user from voice channel.")
+		}
+	}
+	return ctx.Reply("stinky")
+}
+
+func disconnectUserFromVC(session *discordgo.Session, guildID, userID string) error {
+	err := session.GuildMemberMove(guildID, userID, nil)
+	if err != nil {
+		log.Printf("Failed to disconnect user %s from VC: %v", userID, err)
+		return err
+	}
+
+	log.Printf("User %s has been disconnected from voice.", userID)
+	return nil
+}
+
+func monitorVoiceActivity(session *discordgo.Session, guildID, channelID string) {
+	ticker := time.NewTicker(1 * time.Minute)
+	defer ticker.Stop()
```

Comment:
You don't really need this (see the note in [the docs](https://pkg.go.dev/time#NewTicker)).